### DISCUSSION TITLE
Just for fun: drop your favorite developer memes here! 🧵

### DISCUSSION THREAD

**Discussion by lclindeman** [community/community#74905] [New to GitHub]
Let's have some laughs on Day 2 of Universe 2023. Share your favorite developer memes in this thread. I'll start. 🔽 

![computer-science](https://github.com/community/community/assets/3505798/ac29a99a-ecde-4d95-a061-a2b0573f7786)

**Comment by jricciardi** [comment]
![image](https://github.com/community/community/assets/31521880/b4b86c5a-20af-4dd8-9190-9343f12e4cae)

**Reply by lclindeman** [reply]
Could also apply to: DIY home projects and craft projects. 😭

**Reply by jricciardi** [reply]
If I had a dollar for every home project that I started because I thought it would be asy... I could probably pay a professional to do one of those home projects 🤣

**Reply by gaurav-ketkar** [reply]
Love this one - when I first saw it, I was in splits! 
![image](https://github.com/community/community/assets/9720857/4fca41d5-5344-477d-8a5e-26719d390099)

**Reply by mishmanners** [reply]
This is def one of my favourites that I only recently discovered!

**Comment by waldnzwrld** [comment]
![52955598](https://github.com/community/community/assets/4338862/8e1776c0-bb51-41df-9580-1dff086db537)

**Comment by ghostinhershell** [comment]
![image](https://github.com/community/community/assets/14021469/235e143f-dea8-4b01-972b-90b9764fbe9d)

**Comment by soumya03Surya** [comment]
![Review code](https://github.com/community/community/assets/103785216/f54849b5-6e31-4247-952f-7467d38556f9)

**Reply by jricciardi** [reply]
![image](https://github.com/community/community/assets/31521880/75d68c56-1f04-453d-bf37-e44ec8f1f1ba)

**Comment by waldnzwrld** [comment]
![18rtkvbhsv801](https://github.com/community/community/assets/4338862/d4d19c2f-7214-4e5f-b96e-dbb20556a66e)

**Comment by dylan-smith** [comment]
![image](https://github.com/community/community/assets/1508559/ceeb9740-95a9-48fe-b936-f910741d1ba9)

**Reply by gaurav-ketkar** [reply]
I was going to say - a thread about memes and no XKCD yet?

**Reply by jricciardi** [reply]
^

**Comment by jennysharps** [comment]
![](https://miro.medium.com/v2/resize:fit:720/format:webp/1*zFDTdDonxDR5Rh-2rSDo9w.png)

**Reply by jricciardi** [reply]
I used to work with a senior dev named Josh who was at the company for 9 years... he would always complain about running into "Josh code" he had written back in the day that he had to refactor 😆

**Comment by srvchawla** [comment]
![60d33bf3a2121cc74f72a286_0AEnnZYBAYzmSXzyGaopjvEGKKO1yFqf_zMXqFTAT6vUDfXVPRifI3J8oBjZUl51PnRQqf2tqpjiRt33IKgg4sTiJi2YShGTN5iNUIoYbNj2HXobiQg4-k7yBiNT54EKOGYyZOR2](https://github.com/community/community/assets/15349186/3c211ef7-bd5d-4265-90be-161eceb4e194)

**Comment by jricciardi** [comment]
Meme from the live stream 🤣 

![Screenshot 2023-11-09 at 10 25 52 AM](https://github.com/community/community/assets/31521880/cbfb0223-68be-4c4b-b3ad-cefc3404703c)

**Comment by jennysharps** [comment]
![60d33be7eedf8e1f31aabcec_BwENfmI0CU5dZGYlSyo142mpfG08-rYgTS-Qm47uMUXN6JXtmdZvtzVzTooUQdXTWmTD8uzF9N6XQJA2vUIMi53tunFyVtvOBJTNfOjHit2P_JkTmFzFsK7ep6Vb9781XZnRAryH](https://github.com/community/community/assets/1467161/c7f0e6af-d328-4a5b-8f19-cae3c200ace0)

**Reply by jricciardi** [reply]
🤣

**Comment by starspace1** [comment]
![2021-01-15 22 19 22](https://github.com/community/community/assets/1656401/731715e3-0d90-49ab-9dfa-6fb81c7e266d)

![2021-08-25_13-42-15](https://github.com/community/community/assets/1656401/266c19ca-4746-46e9-9a2b-fa320e65dc45)

![FF8iE43XIAwqBGW](https://github.com/community/community/assets/1656401/648bcc4f-992d-496c-89ad-ecd441ba684a)

![IMG_4098](https://github.com/community/community/assets/1656401/9ac9d6f4-daa6-4d44-aa57-b8d18f91f9d9)

![tech debt](https://github.com/community/community/assets/1656401/0db03899-e44a-4a4f-bbc8-19563e5a02d6)

![FvmSPrAWIAM-ZiZ](https://github.com/community/community/assets/1656401/96f21cc4-52c8-453b-b9ba-d71a96f712b5)

**Comment by mishmanners** [comment]
Lols, these are brilliant! Keep them coming people ❤️

**Comment by Vishalk91-4** [comment]
![0_tmfbLDU_hIeg0B3B](https://github.com/community/community/assets/116670999/a78aeb5c-a808-43cc-bdaf-c4a41f669f66)

![funniest-programming-memes-from-this-week-october-14-2023-9](https://github.com/community/community/assets/116670999/4eda6871-7fee-4ec2-a561-64beecc44738)

**Comment by nadyakant** [comment]
![inbox_3486433_9d6c8c9e3341cdab9181307e2f210165_1](https://github.com/community/community/assets/155651297/939a69d6-e4f5-4d65-9043-b0d2eb9a6f28)

**Comment by Akash1134** [comment]
Thank you everyone for contributing to this discussion  and sharing all these developer memes 😆  This particular discussion was a part of Universe 2023 that we are now moving over to other relevant category -- moving this thread over to the `New to GitHub` category. We'd love to hear more such memes from you 🙌
